templates/app-settings.js

```diff
@@ -0,0 +1,8 @@
+AppSettings = {
```

Comment:
@roberto @tayanefernandes We discussed a few ways to change variables in the browser depending on environment, we ended up with the following options:
* Use real google maps api url in code, change etc/hosts file to point it to proxy while in development mode. 
  * This could work for this case but we are not sure about how hard it would be to maintain it since it's an added manual step and in the future we might have to change other settings based on environment.
* Use the window object to check what environment the application is running and change the url accordingly.
  * We are not sure how the app runs in mobile apps, would there be a window object? Would the host still be localhost?
* Use Bash Script to do the same thing gulp is doing right now, renaming a prod or dev javascript file and putting it in www/js folder
  * Personally, I still prefer gulp approach since we won't have cross-platform problems related to BASH (Windows for development??), it's a clear dependency to our project since it's on package.json and I feel more comfortable with gulp's API.